activestorage/app/models/active_storage/blob.rb

```diff
@@ -44,21 +44,25 @@ def find_signed(id)
     end
 
     # Returns a new, unsaved blob instance after the +io+ has been uploaded to the service.
-    def build_after_upload(io:, filename:, content_type: nil, metadata: nil)
+    # Set the +content_type+ and +identify+ to +false+ to manually
```

Comment:
The first part of this sentence makes it sound like both `content_type` and `identify` should be `nil`. Here’s a clarifying edit:

> When providing a content type, pass +identify: false+ to bypass automatic content type inference.

Reply:
Clarified this sentence with the edit bad8fa0